velocity/src/main/java/me/leoko/advancedban/velocity/VelocityMain.java

```diff
@@ -0,0 +1,52 @@
+package me.leoko.advancedban.velocity;
+
+import com.google.inject.Inject;
+import com.velocitypowered.api.event.proxy.ProxyInitializeEvent;
+import com.velocitypowered.api.event.Subscribe;
+import com.velocitypowered.api.event.proxy.ProxyShutdownEvent;
+import com.velocitypowered.api.plugin.Plugin;
+import com.velocitypowered.api.plugin.annotation.DataDirectory;
+import com.velocitypowered.api.proxy.ProxyServer;
+import me.leoko.advancedban.Universal;
+import me.leoko.advancedban.velocity.listener.ChatListenerVelocity;
+import me.leoko.advancedban.velocity.listener.ConnectionListenerVelocity;
+
+import java.nio.file.Path;
+
+@Plugin(
+    id = "advancedban",
+    name = "AdvancedBan",
+    version = "${project.version}"
```

Comment:
I believe this can be avoided by changing the version during compilation as Velocity generates a velocity-plugin.json - Do you know of a way to filter this file while package is being ran?

Reply:
The velocity-plugin.json is generated to target/classes/. It would not be a good idea to make that output folder a resource directory.

Reply:
yeah you're right - I'm not gonna worry about this too much at the minute as it's a pretty minor thing to fix

Reply:
Fixed in next commit